components/ProductMatrix.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <div v-if="error">{{ error }}</div>
+    <div class="product-matrix">
+      <div v-for="card in cards" :key="card.sortKey">
+        <component :is="components[card.type]" :product="card" />
+      </div>
+    </div>
+    <v-row align="center">
+      <v-col class="text-center" cols="12">
+        <v-btn v-if="cards.length <= 6" :loading="loading" @click="loadMore">
+          Visa mer
+        </v-btn>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+/* eslint-disable dot-notation */
+import Vue from 'vue'
+import { mapGetters, mapActions } from 'vuex'
+import ProductCard from './ProductCard.vue'
+import InfoCard from './InfoCard.vue'
+import { CardTypes } from '~/types/Card'
+
+export default Vue.extend({
+  components: {
+    ProductCard
+  },
+  data() {
+    return {
+      error: '',
+      components: {
+        [CardTypes.PRODUCT]: ProductCard,
+        [CardTypes.INFO]: InfoCard
+      }
+    }
+  },
+  computed: mapGetters({
+    cards: 'cards/getCards',
+    loading: 'products/isLoading'
+  }),
+  mounted() {
+    this.init()
```

Comment:
I'm not sure where the best place is, I guess the higher up the tree is better. I was trying to find if we could have a lifecycle hook in the store but I couldn't find anything

> Is there an async way of getting the settings maybe?

Not sure what you mean here, can you clarify?

Reply:
Hm, I was thinking that we should get window.ACC.config when they are availabe, because on coop.se I don't know when they are set and if we fetch them early they might not exist (but they probably will).

I guess these settings should only be set in debug "mode" instead, but not on coop.se when other settings is available.